test/integration/features/basic.feature

```diff
@@ -8,6 +8,14 @@ Feature: Basic
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
 
```

Comment:
It is probably better to actually capture this in a new function that describes more in detail what happens... and why this output. but for the sake of the PR, it might be an idea to just add a comment above:

```gherkin
  # Check if 
  When executing "oc ..."
```